src/components/DayPickerKeyboardShortcuts.jsx

```diff
@@ -176,34 +179,43 @@ class DayPickerKeyboardShortcuts extends React.PureComponent {
 
     return (
       <div>
-        <button
-          ref={this.setShowKeyboardShortcutsButtonRef}
-          {...css(
-            styles.DayPickerKeyboardShortcuts_buttonReset,
-            styles.DayPickerKeyboardShortcuts_show,
-            bottomRight && styles.DayPickerKeyboardShortcuts_show__bottomRight,
-            topRight && styles.DayPickerKeyboardShortcuts_show__topRight,
-            topLeft && styles.DayPickerKeyboardShortcuts_show__topLeft,
-          )}
-          type="button"
-          aria-label={toggleButtonText}
-          onClick={this.onShowKeyboardShortcutsButtonClick}
-          onMouseUp={(e) => {
-            e.currentTarget.blur();
-          }}
-        >
-          <span
+        {renderKeyboardShortcutsButton
+          && renderKeyboardShortcutsButton({
+            // passing in context-specific props
+            ref: this.setShowKeyboardShortcutsButtonRef,
+            onClick: this.onShowKeyboardShortcutsButtonClick,
+            ariaLabel: toggleButtonText,
+          })}
+        {renderKeyboardShortcutsButton
+          || (
```

Comment:
the `|| (` should be on the previous line; there’s no need to have a line break before it